packages/react/src/internal/field-label/FieldLabel.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC, ReactNode } from 'react';
+
+import styles from './FieldLabel.module.css';
+import RequiredIndicator from '../required-indicator/RequiredIndicator';
+
+type FieldLabelProps = {
+  hideLabel?: boolean;
```

Comment:
Would `hidden`be a better prop name here?

Reply:
Sure would!